packages/ignite-animatable/index.js

```diff
@@ -0,0 +1,33 @@
+const NPM_MODULE_NAME = 'react-native-animatable'
+const EXAMPLE_FILE = 'animatableExample.js'
+
+/**
+ * Add ourself to the project.
+ */
+const add = async function (context) {
+  const { ignite } = context
+
+  // install a npm module
+  await ignite.addModule(NPM_MODULE_NAME)
+
+  // copy the example file (if examples are turned on)
+  await ignite.addComponentExample(EXAMPLE_FILE, { title: 'Animatable' })
```

Comment:
I think we only have [addComponentExample](https://github.com/infinitered/ignite/blob/next/packages/ignite/src/plugins/default/extensions/ignite.js#L178) right now.